src/tray_library/Sidebar.tsx

```diff
@@ -152,8 +152,9 @@ export default function Sidebar(props: SidebarProps) {
             setCategory([]);
         }
 
-        setComponentList(response_1);
-        setCategory(response_2);
+        // @ts-ignore
```

Comment:
I think it is recommended to have a short text explaining why the ignore makes sens for the following line.
Could you write a few word why ?